### DISCUSSION TITLE
Follow-Up on Birthday Feature Request

### DISCUSSION THREAD

**Discussion by ddecodes** [BitByte-TPC/alumni#147] [Q&A]
Dear Maintainer,

I hope this message finds you well. I am writing to follow up on our previous discussion in the introductory session of the alumni connect website regarding the potential addition of a birthday feature to the website.

During our last conversation, we touched on the idea of implementing a feature that allows users to input and track birthdays within the application, possibly including notifications or reminders for upcoming birthdays, as well as the option to display birthday information in user profiles.

I wanted to inquire if there has been any further consideration or decision regarding this feature. Could you please provide an update on the current status or any thoughts you might have on its feasibility and potential implementation?

If provided a chance, I am happy to assist in any way possible to help with the implementation process.

Thank you  for your time and all the effort you dedicate to maintaining and enhancing this project. I look forward to your response.

**Comment by Namanv0509** [comment]
Respected Maintainer , as mentioned by ddecodes , the feature of Birthday wisher was to be included , if further details on it can be provided at earliest would be really helpful .
Thank you .
